Octokit/Helpers/Ensure.cs

```diff
@@ -49,16 +50,16 @@ public static void GreaterThanZero([ValidatedNotNull]TimeSpan value, string name
         }
 
         /// <summary>
-        /// Checks an array argument to ensure it isn't null or empty.
+        /// Checks a list argument to ensure it isn't null or empty.
         /// </summary>
         /// <param name = "value">The argument value to check</param>
         /// <param name = "name">The name of the argument</param>
-        public static void ArgumentNotNullOrEmptyArray<T>([ValidatedNotNull]T[] value, string name)
+        public static void ArgumentNotNullOrEmptyList<T>([ValidatedNotNull]IReadOnlyList<T> value, string name)
```

Comment:
this method could probably just take an `IEnumerable<T>` rather than `IReadOnlyList<T>` and would then be more reusable for checking other potential lists in the future